src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, FigureDraw {
```

Comment:
Make sure to not put all behavior into a single interface if the methods are conceptually different. In this case, `AreaCalculator` and `FigureDraw` should be separate if their methods are not conceptually related.